openedx/features/idea/models.py

```diff
@@ -62,6 +61,16 @@ class Idea(OrganizationBase, Location, VisualAttachment):
     overview = models.CharField(max_length=OVERVIEW_MAX_LENGTH)
     description = models.TextField()
     implementation = models.TextField(blank=True)
+    favorites = models.ManyToManyField(User, related_name='favorited')
+
+    def toggle_favorite(self, user):
```

Comment:
This maybe more related to coding style but here's my take if you like it maybe you can incorporate this change:

```
    def toggle_favorite(self, user):
        if self.favorites.filter(pk=user.id).exists():
            self.favorites.remove(user)
            return False

        self.favorites.add(user)
        return True
```